src/backend/utils/mb/conversion_procs/euc_tw_and_big5/big5.c

```diff
@@ -370,6 +370,7 @@ CNStoBIG5(unsigned short cns, unsigned char lc)
 				if (b1c4[i][1] == cns)
 					return b1c4[i][0];
 			}
+			break;
```

Comment:
I suggest to remove `default: break;` and don't add `break`. The code will be simpler

Reply:
Done